core/emulator.h

```diff
@@ -39,8 +39,10 @@ void dc_request_reset();
 void dc_exit();
 void dc_resume();
 void dc_step();
-void dc_savestate();
-void dc_loadstate();
+void dc_savestate(int index = 0);
+void dc_savestate(int index);
```

Comment:
I'm surprised the compiler isn't complaining about this redefinition. Anyway this line isn't necessary.